src/Tribe/Capabilities.php

```diff
@@ -15,24 +46,62 @@ class Tribe__Events__Capabilities {
 			'delete_private_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'author' => array( // full permissions for content the user created
+		],
+		'author' => [ // full permissions for content the user created
 			'read',
 			'edit_posts',
 			'edit_published_posts',
 			'delete_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'contributor' => array( // create, but not publish
+		],
+		'contributor' => [ // create, but not publish
 			'read',
 			'edit_posts',
 			'delete_posts',
-		),
-		'subscriber' => array( // read only
+		],
+		'subscriber' => [ // read only
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 2 space(s) between "'subscriber'" and double arrow, but found 1.
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 8b253b5172dc6fd746959d8e36b56409</sub>